microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -970,6 +970,33 @@ WebView2 provides functionality to handle the JavaScript function `window.open()
 
 ---
 
+
+#### todo: add link to the above tab-set?  Nov 2023 incoming
+
+* Added source frame info to the `NewWindowRequested` event arguments, to identify the source of the request:
```

Comment:
If we are not calling out other properties from the NewWindowRequestedEventArgs, we don't need to explicitly call the OriginalSourceFrameInfo either.

Reply:
removed temp section